packages/webchat/src/main.tsx

```diff
@@ -174,7 +174,16 @@ class Web extends React.Component<MainProps> {
       case 'configure':
         return this.props.updateConfig!(Object.assign({}, constants.DEFAULT_CONFIG, data.payload))
       case 'mergeConfig':
-        return this.props.mergeConfig!(data.payload)
+        this.props.mergeConfig!(data.payload)
+
+        const oldUserId = this.socket.socket.userId
+        await this.socket.reload(data.payload)
```

Comment:
We only want to reconnect when **some** of the config changed, not for all of them

Reply:
I made it so the reload function of the socket checks if reconnection is needed